packages/bitcore-node/src/modules/ethereum/p2p/p2p.ts

```diff
@@ -116,13 +126,13 @@ export class EthP2pWorker extends BaseP2PWorker<IEthBlock> {
     let firstConnect = true;
     let connected = false;
     let disconnected = false;
-    const { host, port } = this.chainConfig.provider;
+    const { host, port } = this.chainConfig.provider || this.chainConfig.providers![0];
     while (!this.disconnecting && !this.stopping) {
       try {
         if (!this.web3) {
           const { web3 } = await this.getWeb3();
           this.web3 = web3;
-          this.rpc = new ParityRPC(this.web3);
+          this.rpc = new Rpcs[this.chainConfig.client || 'erigon'](this.web3);
```

Comment:
Because we say we don't maintain Erigon support, once we have the Geth class in working order we should switch this to default to Geth.